aiocogeo/partial_reads.py

```diff
@@ -280,7 +271,7 @@ def _resample(
         if self._add_mask:
             mask = Image.fromarray(clipped.mask[0,...])
             resized_mask = np.array(mask.resize((out_shape[0], out_shape[1]), resample=Image.BILINEAR))
-            resized_mask = np.stack((resized_mask for _ in range(img_tiles.bands)))
+            resized_mask = np.stack([resized_mask for _ in range(img_tiles.bands)])
```

Comment:
This will prevent this annoying numpy warning:
```aiocogeo/cog.py:367: FutureWarning: arrays to stack must be passed as a "sequence" type such as list or tuple. Support for non-sequence iterables such as generators is deprecated as of NumPy 1.16 and will raise an error in the future.```